platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -101,6 +106,13 @@ public void enableLongPress(boolean enable) {
 	public void enablePanningAndScalingGestures(boolean enable) {
 		this.godotGestureHandler.setPanningAndScalingEnabled(enable);
 	}
+	
+	/**
+	 * Set RotaryInput axis. This is Y by default.
+	 */
+	public void setRotaryInputAxis(int axis) {
+    	rotaryInputAxis = axis;
+    }
```

Comment:
Looks like there is an issue with indentation here.

Reply:
Fixed. And All others indentation errors.